src/server/debug/server/server.go

```diff
@@ -347,6 +355,9 @@ func redirectBinaryFunc(ctx context.Context) redirectFunc {
 }
 
 func (s *debugServer) Dump(request *debug.DumpRequest, server debug.Debug_DumpServer) error {
+	if request.Limit == 0 {
```

Comment:
drive-by comment: I feel like `0` should be zero commits and `-1` should be unlimited, otherwise there is no way to specify `0`. but I suppose golang default values make this difficult since we want to default behavior to be unlimited

Reply:
Yeah, I don't really have a strong opinion either way. I was aiming to be consistent with list commit which interprets 0 to mean all of them and it is nice to be able to use the Go default behavior, but like you said we cannot specify 0 with this. What do you think @jdoliner ?

Reply:
I think having 0 mean all of them is probably easiest. I don't think there's much need to be able to do a dump that doesn't get any commits or jobs. The need seems small enough that we should just do the easy thing.